src/components/pages/Categories/List.vue

```diff
@@ -69,6 +79,19 @@ export default {
     updateValue(event) {
       this.$store.dispatch('categories/updateCategories', event.target.value);
     },
+    openModal(categoryId, categoryName) {
+      const payload = {
+        categoryId,
+        categoryName,
+      };
+      this.$store.dispatch('categories/confirmDeleteCategory', payload);
```

Comment:
[imo]

一時変数は逆に可読性を落としてしまうので、以下のような記述で良いかなと思います

```suggestion
      this.$store.dispatch('categories/confirmDeleteCategory',  { categoryId, categoryName );
```